chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/surveillance/reviewer/ComparisonReviewer.java

```diff
@@ -1,8 +1,19 @@
 package gov.healthit.chpl.validation.surveillance.reviewer;
 
 import gov.healthit.chpl.domain.surveillance.Surveillance;
+import gov.healthit.chpl.domain.surveillance.SurveillanceNonconformity;
+import gov.healthit.chpl.domain.surveillance.SurveillanceRequirement;
+import gov.healthit.chpl.util.SurveillanceUtil;
 
-public interface ComparisonReviewer {
+public abstract class ComparisonReviewer {
```

Comment:
Maybe you could have used a default method instead of converting to an abstract class. Nothing wrong with this way, just a thought.